AV/VehicleAutomationLevels.rdf

```diff
@@ -153,16 +155,16 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">ADS dedicated vehicle</rdfs:label>
-		<skos:definition xml:lang="en">vehicle designed to be operated exclusively by a level 4 or level 5 ADS for all trips within its given ODD limitations (if any)</skos:definition>
+		<skos:definition xml:lang="en">ADS-equipped vehicle designed for driverless operation under routine/normal operating conditions during all trips within its given ODD (if any)</skos:definition>
 		<skos:example xml:lang="en">A level 4 ADS-DV designed to operate exclusively within a corporate campus where it picks up and discharges passengers along a specific route specified by the ADS-DV dispatcher.</skos:example>
 		<skos:example xml:lang="en">A level 4 ADS-DV designed to operate exclusively within a geographically prescribed central business district where it delivers supplies using roads (but not necessarily routes) specified by the ADS-DV dispatcher.</skos:example>
 		<skos:example xml:lang="en">A level 5 ADS-DV capable of operating on all mapped roads in the US that are navigable by a human driver. The user simply inputs a destination, and the ADS-DV automatically navigates to that destination.</skos:example>
 		<AnnotationVocabulary:abbreviation>ADS-DV</AnnotationVocabulary:abbreviation>
-		<AnnotationVocabulary:adaptedFrom>SAE-J3016 (2018), sec. 3.3</AnnotationVocabulary:adaptedFrom>
+		<AnnotationVocabulary:adaptedFrom>SAE-J3016 APR2021, sec. 3.32.3</AnnotationVocabulary:adaptedFrom>
+		<AnnotationVocabulary:adaptedFrom>SAE-J3016 JUN2018, sec. 3.3</AnnotationVocabulary:adaptedFrom>
```

Comment:
If the description of ADS dedicated vehicle comes from APRI2021 version, then the reference to JUN2018 is obsolete. This applies to all such references in this commit.